server/src/test/java/org/elasticsearch/search/internal/ContextIndexSearcherTests.java

```diff
@@ -232,6 +244,12 @@ public void onRemoval(ShardId shardId, Accountable accountable) {
 
         // make sure scorers are created only once, see #1725
         assertEquals(1, searcher.count(new CreateScorerOnceQuery(new MatchAllDocsQuery())));
+
+        TopDocs topDocs = searcher.search(new BoostQuery(new ConstantScoreQuery(new TermQuery(new Term("foo", "bar"))), 3f), 1);
+        assertEquals(topDocs.totalHits.value, 1);
+        assertEquals(topDocs.scoreDocs.length, 1);
+        assertEquals(topDocs.scoreDocs[0].score, 3f, 0);
```

Comment:
can you put the expected value (ie. `1` and `3f`) first in order to get better error messages in case of failure?